### ISSUE TITLE
Random Freezes with "Select oogabooga" in title bar

### ISSUE THREAD

**Issue by Kippykip** [oobabooga/text-generation-webui#2994]
### Describe the bug

Sometimes everything will freeze up, especially when using the API extension.
But this has also happened when I'm saving characters and stuff too sometimes.

Basically text-generation-ui will randomly freeze, and the CMD window's titlebar will change with **Select oogabooga** in the title.
Pressing CTRL+C or pressing Y usually fixes it. But it doesn't show what the reasoning is.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

It's intermittent, but seems to happen the most when loading 30b models through the API then talking through TavernUI in my experience.
It has also happened to me when saving characters in the text-generation-ui a few times too.

### Screenshot

![image_2023-07-04_100756398](https://github.com/oobabooga/text-generation-webui/assets/33023808/3ba75870-4fad-4a9f-9ed8-d13f65eb0c66)

### Logs

```shell
Command line hangs during this with no message.
```

### System Info

```shell
Windows 10 Enterprise LTSC
Nvidia GTX 3090
```

**Comment by missionfloyd** [comment]
You've clicked in the console window. Press escape.

**Comment by missionfloyd** [comment]
Perhaps something about the remote desktop is making it act strange, like clicking the wrong window or in the wrong spot.

Could it be lagging? i.e., you click on something but the image on your monitor hasn't updated yet, so you end up clicking something else.

Could you avoid it my minimizing the console window?

**Comment by Kippykip** [comment]
> Perhaps something about the remote desktop is making it act strange, like clicking the wrong window or in the wrong spot.
> 
> Could it be lagging? i.e., you click on something but the image on your monitor hasn't updated yet, so you end up clicking something else.
> 
> Could you avoid it my minimizing the console window?

Yeah I forgot to post an update but that's my workaround for now, I started minimising before disconnecting my remote desktop session and I haven't had the problem since. I'm still not sure why it's happening though
